### DISCUSSION TITLE
How to Forward certain directories created under **/usr/local/antmedia/** to default port 443

### DISCUSSION THREAD

**Discussion by AppThisThing** [ant-media/Ant-Media-Server#5178] [Q&A]
I want to remove the port number "5443" from the URL, Only for Ant Media App-Name.

From: https://DOMAIN:5443/App-Name
To: https://DOMAIN/App-Name/

The goal: Not to affect or make a difference when accessing Non AMS App-Name URLs.
so when I go to https://DOMAIN/url_Here “I still see” Hello World.
so when I go to https://DOMAIN/url_There “I still see” Hello World 2.

- I only wish to direct port 5443 to Ant Media folders.
- It sounds a little crazy, - only certain directories created under **/usr/local/antmedia/** should be Forward to default port 443

I tried this solution: (it forwards all, not just AMS folders.)
**Forward default http (80), https (443) ports to 5080 and 5443**
https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/#forward-default-http-80-https-443-ports-to-5080-and-5443

How can I customize this a bit, is it do able?

Thanks

**Comment by Mohit-3196** [comment]
Hi @AppThisThing ,
Sorry, but it is a little unclear to me as to the use case and what exactly you would like to achieve. Is it that you want to access the Web Panel on 5443 and other things like sample publish and play on 443?
Can you give me some more information?

Also, this is expected. The port forwarding is applied to the server and not just some specific section of the service.
Also, this directory /usr/local/antmedia is the default installation directory of AMS.
